### PR TITLE
Feature/custom calender front end integration

### PR DESCRIPTION
**Pull request by nolantellis**
* First Phase of Custom Calendar

### CODE REVIEW

File: Apromore-Calendar/src/test/java/org/apromore/calendar/service/DurationCalculationUnixTsLoadUnitTest.java [hunk 1]
```diff
@@ -49,11 +50,11 @@
 @Ignore
```
Comment (nolantellis) [inline Apromore-Calendar/src/test/java/org/apromore/calendar/service/DurationCalculationUnixTsLoadUnitTest.java]: Add comment why this test is ignored.

File: Apromore-Database/src/main/java/org/apromore/dao/model/Holiday.java [hunk 2]
```diff
@@ -50,6 +52,8 @@
 	private Long id;
 
 	private Long referenceId;
+	
+	private HOLIDAYTYPE holidayType=HOLIDAYTYPE.PUBLIC;
```
Comment (nolantellis) [inline Apromore-Database/src/main/java/org/apromore/dao/model/Holiday.java]: space between sign

File: Apromore-Calendar/src/main/java/org/apromore/calendar/service/CalendarService.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+import java.util.List;
+import org.apromore.calendar.exception.CalendarAlreadyExistsException;
+import org.apromore.calendar.exception.CalendarNotExistsException;
+import org.apromore.calendar.model.CalendarModel;
+import org.apromore.calendar.model.HolidayModel;
+import org.apromore.calendar.model.WorkDayModel;
+import org.apromore.dao.model.CustomCalendar;
+
+public interface CalendarService {
```
Comment (franklma) [inline Apromore-Calendar/src/main/java/org/apromore/calendar/service/CalendarService.java:32]: Modifiers 'final' and 'public' are redundant for interface

**Review by franklma — APPROVED** [review]

**Review by dreamind — APPROVED** [review]

**Review by raboczi — APPROVED** [review]
This builds and starts and I'm satisfied it makes sense following the walk-through.  I committed some superficial tweaks to the doc comments.

I question whether site.properties.template should be added again, but that's not a sufficient issue to block the merge.

File: Apromore-Calendar/src/main/java/org/apromore/calendar/service/CalendarService.java (hunk 3 above)
Comment (nolantellis) [reply Apromore-Calendar/src/main/java/org/apromore/calendar/service/CalendarService.java:32]: lol. Yup. They get aiutogenerated by the IDE

**Pull request merged**